src/Alert/Alert.tsx

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { css } from 'styled-components';
-import { space } from 'styled-system';
-import propTypes from '@styled-system/prop-types';
+import { space, SpaceProps } from 'styled-system';
 import { getComponentVariant, createComponent } from '../utils';
 
-const StyledAlert = createComponent({
+export interface AlertProps extends SpaceProps {
```

Comment:
Looking through a couple related PRs, I'm seeing discrepancies around defining props.  Here it is an interface, elsewhere I've seen type.

Reply:
hmm, I normally use interface to define objects.  Is there a reason why we would use type over interfaces for objects?